tests/fast-integration/skr-test/README.md

```diff
@@ -0,0 +1,92 @@
+# SKR test
+
+This test covers SKR (SAP Kyma Runtime).
+
+## File Structure consists of the following parts:
+- **provision** folder contains the scripts for provisioning and de-provisioning the SKR cluster using KEB client
+- **oidc** folder contains the OIDC related tests
+- **commerce-mock** folder consists of two scripts:
+  1. **index.js** contains the commerce mock tests suite
+  2. **prep.js** can be called in stand-alone mode for commerce mock resources preparation
+
+##Run the SKR test
+
+Before you run the test make sure you prepare the `.env` file based on the `.env.template`:
+```
+INSTANCE_ID
+SKIP_PROVISIONING
+KEB_HOST=
+KEB_CLIENT_ID=
+KEB_CLIENT_SECRET=
+KEB_GLOBALACCOUNT_ID=
+KEB_SUBACCOUNT_ID=
+KEB_USER_ID=
+KEB_PLAN_ID=
+GARDENER_KUBECONFIG=
+
+COMPASS_HOST=
+COMPASS_CLIENT_ID=
+COMPASS_CLIENT_SECRET=
+COMPASS_TENANT=
+
+KCP_KEB_API_URL=
+KCP_GARDENER_NAMESPACE=
+KCP_OIDC_ISSUER_URL=
+KCP_OIDC_CLIENT_ID=
+KCP_OIDC_CLIENT_SECRET=
+KCP_TECH_USER_LOGIN=
+KCP_TECH_USER_PASSWORD=
+KCP_MOTHERSHIP_API_URL=
+KCP_KUBECONFIG_API_URL=
+
+AL_SERVICE_KEY= #must be a cloud foundry service key with the info about `UAA (User Account and Authentication)`. Learn more about [Managing Service Keys in Cloud Foundry](https://docs.cloudfoundry.org/devguide/services/service-keys.html).
+```
+
+Run the following command to set up the environment variables in your system:
+
+```bash
+export $(xargs < .env)
+```
+### Usage:
+
+The SKR test can be used in two modes:
+
+#### With provisioning
+In this case the tests executes the following steps:
+   1. Provision SKR cluster and register compass resources
+   2. OIDC Test
+   3. Commerce Mock Test
+   4. De-provision the SKR instance and clean the compass resources
+
+Run the test y calling the following target:
+```bash
+npm run test-skr
+#or
+make test-skr
+```
+
+#### Without Provisioning. 
+In this mode the test will additionally need following environment variables:
+- `SKIP_PROVISIONING` needs be set to `true` in that case
+- `INSTANCE_ID` the uuid of the provisioned SKR instance
+- `TEST_SUFFIX` is used to determine the compass scenario and the app assigned to it. Will be randomly generated if not set.
+
+>**IMPORTANT:** The `TEST_SUFFIX must be 4 characters long and is required for every commerce-mock test run. In case the compass scenario with the given suffix already exists, the test will try to reuse it.
+
+This mode consists of the following steps:
+1. Ensure SKR exists and register compass resources
+2. OIDC Test
+3. Commerce Mock Test
+4. Clean the compass resources
+
+Run the SKR test in that mode:
+```bash
+make test-skr SKIP_PROVISIONING=true
+#or
+npm run test-skr # when all env vars are exported
+```
+
+#### If you only need to prepare the commerce mock resources run:
```

Comment:
What is this? Initially, we say there are two modes, is this a third?? 

```suggestion
   - If you only need to prepare the commerce mock resources, run:
```

Reply:
This is not a mode, but an additional standalone script, which only install some prerequisites for CommerceMock tests, I mentioned it in the file structure already, I thought it is enough to highlight it.